src/.defaultsForValidator.js

```diff
@@ -106,6 +106,30 @@ const defaults = {
     'schemas': {
       'json_or_param_binary_string': 'warning'
     }
+  },
+  'spectral': {
```

Comment:
Since i'm calling `getConfigObject` if there is no spectral section in the validaterc, it will take the rules from the spectral rules from `.defaultsForValidator.js` and these changes also allow the `init` command to generate the new spectral section